projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/LazyToTruffleConverterImpl.java

```diff
@@ -109,21 +117,83 @@ public RootCallTarget convert() {
                         method.getParameters().size(), symbols,
                         method, liveness, notNullable, dbgInfoHandler);
         method.accept(visitor);
+
+        LLVMControlFlowGraph cfg = new LLVMControlFlowGraph(method.getBlocks().toArray(new InstructionBlock[method.getBlocks().size()]));
```

Comment:
It is not necessary to pass a preallocated array to `toArray`. Its safer to instead use a constant empty array like `FunctionDefinition.EMPTY`.